oxidation/bindings/generator/api.py

```diff
@@ -25,35 +25,63 @@ class Ref(Generic[TypeVar("REFERENCED")]):  # noqa
 
 # A type that should be converted from/into string
 class StrBasedType:
-    pass
+    ...
 
 
 class OrganizationID(StrBasedType):
-    pass
+    ...
```

Comment:
same as above, use pass for everything except function definitions

Reply:
Done